core/src/main/java/org/elasticsearch/search/SearchModule.java

```diff
@@ -615,7 +616,11 @@ public void register(String name, Suggester<?> t) {
         highlighters.register("fvh",  new FastVectorHighlighter(settings));
         highlighters.register("plain", new PlainHighlighter());
         highlighters.register("postings", new PostingsHighlighter());
-
+        UnifiedHighlighter uh = new UnifiedHighlighter();
+        highlighters.register("unified", uh);
+        highlighters.register("unified_plain", uh);
+        highlighters.register("unified_postings", uh);
+        highlighters.register("unified_fvh", uh);
```

Comment:
The experimental highlighter used another field if you were going to override. It called it `"hit_source"` I think. I like that kind of thing better because it makes it more clear that you are only changing where the hits are calculated from not the actual highlighter implementation.